README.md

```diff
@@ -112,100 +124,7 @@ $hashtableData = Read-CloudEventJsonData -CloudEvent $cloudEvent
 ```powershell
 $hashtableData = Read-CloudEventXmlData -CloudEvent $cloudEvent -ConvertMode SkipAttributes
```

Comment:
Can you please explain SkipAttr flag and show a command to retrieve all the options the module has, like a detailed module README section for the parameters you nicely documented?

Reply:
Good point it is good to be more clear what the parameter purpose is for. Added information about the parameter with reference to the cmdlet help. I don't want the README to be the complete help of the SDK. I prefer to keep it concise and leave all the details in the cmdlets help